libmscore/scorefile.cpp

```diff
@@ -433,25 +435,26 @@ bool MasterScore::saveFile(bool generateBackup)
             // remove old backup file if exists
             // remove the backup file in the same dir as score (the traditional place) if exists
             //
-            QString backupDirString = info.path() + QString(QDir::separator()) + ".backup";
+            const QString backupSubdirString = preferences.getString(PREF_APP_BACKUP_SUBFOLDER);
+            const QString backupDirString = info.path() + QString(QDir::separator()) + backupSubdirString;
             QDir backupDir(backupDirString);
             if (!backupDir.exists()) {
-                  dir.mkdir(".backup");
+                  dir.mkdir(backupSubdirString);
 #ifdef Q_OS_WIN
-                  QString backupDirNativePath = QDir::toNativeSeparators(backupDirString);
+                  const QString backupDirNativePath = QDir::toNativeSeparators(backupDirString);
 #if (defined (_MSCVER) || defined (_MSC_VER))
    #if (defined (UNICODE))
                   SetFileAttributes((LPCTSTR)backupDirNativePath.unicode(), FILE_ATTRIBUTE_HIDDEN);
    #else
                   // Use byte-based Windows function
                   SetFileAttributes((LPCTSTR)backupDirNativePath.toLocal8Bit(), FILE_ATTRIBUTE_HIDDEN);
-   #endif
```

Comment:
If `unnesting` this, do it for the if and the else too. 
One portability trick BTW is to have the # in the first column, and spaces after that for indentation.